src/main/cpp/src/SparkResourceAdaptorJni.cpp

```diff
@@ -14,18 +14,16 @@
  * limitations under the License.
  */
 
+#include <cudf/logger.hpp>
```

Comment:
This is a semantic change to how logging is performed in this class.  Currently we disable libcudf logging, but that does not disable the logging in this class.  If we leverage the cudf logger here, then we will only support logging in this class if we support logging in libcudf.

Like the faultinj project, I think we need to fetch our own spdlog here and use it as our own logger.